opacus/tests/privacy_engine_test.py

```diff
@@ -74,46 +147,115 @@ def forward(self, x):
     def name(self):
         return "SampleConvNet"
 
+class SampleLSTMs():
+    input_size = 20
+    max_seq_length = 30
+    def __init__(self, model_initializer, **kwargs):
+        self.criterion = nn.MSELoss()
+        self.init_lstms()
+        fn_kwargs = kwargs.copy()
```

Comment:
Its use case is when you are creating a DP version of LSTM. For doing that I am passing a list of ```state_dict``` for all the non-private LSTMs in the kwargs dictionary and since I am individually initializing the private models and their corresponding optimizers, ```fn_kwargs``` only need to have the ```state_dict``` for the corresponding non-private lstm model.

Reply:
Oh, I see
Thanks for explaining